src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/GetExceptionCodeTests.cs

```diff
@@ -3,11 +3,13 @@
 // See the LICENSE file in the project root for more information.
 
 using Xunit;
+using Microsoft.DotNet.XUnitExtensions.Attributes;
 
 #pragma warning disable CS0618 // Type or member is obsolete
 
 namespace System.Runtime.InteropServices.Tests
 {
+    [SkipOnMono("https://github.com/mono/mono/issues/15085 - Marshal Methods WILL NOT BE Implemented in MonoVM")]
```

Comment:
```suggestion
    [SkipOnMono("Marshal.GetExceptionCode will not be implemented in Mono, see https://github.com/mono/mono/issues/15085.")]
```
- You can make the reason more specific (which Marshal methods?)
- The message is more readable if you start with human description first, and add issue link at the end. It is the pattern used for other similar messages in libraries.
- No need to use ALL CAPS

Reply:
(Not commenting on all of these.)

Reply:
Thanks! I'll update it.